examples/with-react-ga/components/Layout.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react'
+import Router from 'next/router'
 import { initGA, logPageView } from '../utils/analytics'
 
+Router.onRouteChangeComplete = (url) => {
+  logPageView()
+}
+
```

Comment:
@timneutkens But how will you log page view for initial render(SSR)? As Router events are fired only on client side routing.